### DISCUSSION TITLE
Failed to load version/info Error: Unauthorized

### DISCUSSION THREAD

**Discussion by weiwendi** [argoproj/argo-workflows#7098] [Q&A] [answered]
time="2021-10-29T11:45:54.025Z" level=info msg="finished unary call with code Unauthenticated" error="rpc error: code = Unauthenticated desc = token not valid for running mode" grpc.code=Unauthenticated 
grpc.method=GetInfo grpc.service=info.InfoService grpc.start_time="2021-10-29T11:45:54Z" grpc.time_ms=0.079 span.kind=server system=grpctime="2021-10-29T11:45:57.971Z" level=info msg="finished unary call with code Unauthenticated" error="rpc error: code = Unauthenticated desc = token not valid for running mode" grpc.code=Unauthenticated 
grpc.method=GetUserInfo grpc.service=info.InfoService grpc.start_time="2021-10-29T11:45:57Z" grpc.time_ms=0.085 span.kind=server system=grpc

**Comment by terrytangyuan** [comment, marked as answer]
Make sure you’ve configured your server correctly and your client provides the correct authentication token (if the “client” auth-mode is used): https://argoproj.github.io/argo-workflows/argo-server-auth-mode/
